Modifier/ProductModifier.php

```diff
@@ -68,18 +69,48 @@ protected function modify(AbstractImportItem $eventItem)
      *
      * @param ProductDocument      $document
      * @param CatalogProductEntity $entity
+     *
+     * @throws ItemSkipException
      */
     protected function transform(ProductDocument $document, CatalogProductEntity $entity)
     {
-        $document->setId($entity->getId());
-        $document->setUrls([]);
-        $document->setExpiredUrls([]);
-        $document->setSku($entity->getSku());
-
-        $this->addPrice($entity, $document);
-        $this->addTextAttributes($entity, $document);
-        $this->addVarcharAttributes($entity, $document);
-        $this->addCategories($entity, $document);
+        if ($this->isProductActive($entity)) {
+            $document->setId($entity->getId());
+            $document->setUrls([]);
+            $document->setExpiredUrls([]);
+            $document->setSku($entity->getSku());
+
+            $this->addPrice($entity, $document);
+            $this->addTextAttributes($entity, $document);
+            $this->addVarcharAttributes($entity, $document);
+            $this->addCategories($entity, $document);
+        } else {
+            throw new ItemSkipException('Product ' . $entity->getId() . ' is not active, so it wont be imported.');
+        }
+    }
+
+    /**
+     * Checks if product is active (check only if product is set as enabled disabled in magento).
```

Comment:
maybe you wanted to write "enabled or disabled", because a bit unclear sentence now.